content/en/docs/ops/common-problems/observability-issues/index.md

```diff
@@ -45,32 +45,6 @@ service. Some common time synchronization systems are NTP and Chrony. This is es
 problematic in engineering labs with firewalls. In these scenarios, NTP may not be configured
 properly to point at the lab-based NTP services.
 
-## Mixer Telemetry Issues
-
-{{< warning >}}
-Mixer is deprecated. The functionality provided by Mixer is being moved into the Envoy proxies.
-Use of Mixer with Istio will only be supported through the 1.7 release of Istio.
-{{</ warning>}}
-
-### Expected metrics are not being collected
-
-The following procedure helps you diagnose problems where metrics
-you are expecting to see reported are not being collected.
-
-The expected flow for metrics is:
-
-1. Envoy reports attributes from requests asynchronously to Mixer in a batch.
-
-1. Mixer translates the attributes into instances based on the operator-provided configuration.
-
-1. Mixer hands the instances to Mixer adapters for processing and backend storage.
-
-1. The backend storage systems record the metrics data.
-
-The Mixer default installations include a Prometheus adapter and the configuration to generate a [default set of metric values](/docs/reference/config/policy-and-telemetry/metrics/) and send them to the Prometheus adapter. The Prometheus adapter configuration enables a Prometheus instance to scrape Mixer for metrics.
-
-If the Istio Dashboard or the Prometheus queries don’t show the expected metrics, any step of the flow above may present an issue. The following sections provide instructions to troubleshoot each step.
-
 #### Verify Istio CNI pods are running (if used)
```

Comment:
This looks strange now with `####` header, when the others are all `##`. This used to be nested under the Mixer section. Is it still a common problem, without Mixer? If so, what about some of the other nested entries, like `Verify Prometheus configuration`?

Reply:
The `Verify Prometheus configuration` depends on `targets: ['istio-mixer.istio-system:42422']` which uses mixer in some way. So it needs to be removed.